microshift_release_notes/microshift-4-13-release-notes.adoc

```diff
@@ -59,8 +59,10 @@ This release adds improvements related to the following components and concepts.
 //[id="microshift-4-13-networking"]
 //=== Networking
 
```

Comment:
This references OSDOCS-556**3**.

Reply:
Fixed the PR and commit message to point to the same ticket